functions/exercise/3_implementAnd.js

```diff
@@ -13,4 +13,33 @@
 
 
 // Sample function : andOperator(null, 1)
-// Expected return value : false
\ No newline at end of file
+// Expected return value : false
+
+
+function andOperator(a,b){
+    if(a === 0){
+        if(b === 1){
+           return false; 
+        } else if(b===0){
+            return false;
+        } else {
+            return false;
```

Comment:
if a is zero then it will result in false no mater what is  value of b , so you can skip the line 21 to 26